receiver.go

```diff
@@ -182,6 +182,8 @@ func (rec *githubactionsannotationsreceiver) processAnnotations(ctx context.Cont
 	resourceAttributes := resourceLogs.Resource().Attributes()
 	serviceName := generateServiceName(rec.config, repository.FullName)
 	resourceAttributes.PutStr("service.name", serviceName)
+	resourceAttributes.PutStr("github.event", "github-actions")
```

Comment:
IMO, we should use something more custom.

Using the existing event attribute may confuse us in the future, as this GitHub event doesn't exist.

Reply:
maybe `github.log`:
- `github.log: annotations`
- `github.log: github-actions`

then eventually:
- `github.log: audit`


-------

Although I see the concept of `event.dataset`, https://github.com/elastic/integrations/tree/main/packages/github uses 
- `"dataset": "github.secret_scanning"`
- `"dataset": "github.dependabot"`
- `"dataset": "github. code_scanning"`
- `"dataset": "github.audit",`


We could use
- `event.dataset: github.annotations`
- `event.dataset: github.actions`

Reply:
I like it.

I was also looking into this and I found this
https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-resource

Looks like the same concept.

Reply:
Ok, do we agree to use the `Resource` attributes for the logs with `event.dataset` and the values ('github.annotations', 'github.actions')?

Reply:
Uh oh, I think I confused something. I somehow thought there would be a field called `resource` in elasticsearch. 

but we are already using resource attributes 😅 

But yes, event.dataset sounds good to me.